modules/core/src/lib/base-attribute.js

```diff
@@ -48,7 +48,7 @@ export default class BaseAttribute {
     if (buffer) {
       this.externalBuffer = buffer;
       this.constant = false;
-      this.value = null;
+      this.value = value || null;
```

Comment:
Is the purpose of this change to make the hack you mentioned in this comment work?
```
transition.attributeInTransition.update({
  buffer,
  // Hack: Float64Array is required for double-precision attributes
  // to generate correct shader attributes
  value: attribute.value
});
```

If so, can we add a comment here, too, since it will seem strange (or like a bug) to other readers (and our future selves when we forget how all this works).